packages/web/src/js/utils/Debounce.ts

```diff
@@ -1,8 +1,19 @@
 export const debounce = <T>(callback: (props: T) => void, delay: number) => {
   let timeout: NodeJS.Timeout;
+  let isScrolling = false;
```

Comment:
I asked chatgpt to suggest better name. This is the result:
- isDebouncing
- debounceInProgress
- isDelayed
- debounceActive
- isWaiting
- delayInProgress
- actionInProgress
- isThrottled

I don't have a strong opinion on any of them, but I would prefer less the waiting or the *InProgress ones.

Reply:
Fixed.